jib-gradle-plugin/README.md

```diff
@@ -174,13 +171,22 @@ Property | Type
 `username` | `String`
 `password` | `String`
 
+<a name="container-closure"></a>`container` is a closure with the following properties:
+
+Property | Type | Default | Description
+--- | --- | --- | ---
+`jvmFlags` | `List<String>` | *None* | Additional flags to pass into the JVM when running your application.
+`mainClass` | `String` | *Inferred\** | The main class to launch your application from.
+`args` | `List<String>` | *None* | Default main method arguments to run your application with.
+`format` | `String` | `Docker` | Use `OCI` to build an [OCI container image](https://www.opencontainers.org/).
+
+
 ### Example
 
 In this configuration, the image is:
 * Built from a base of `openjdk:alpine` (pulled from Docker Hub)
 * Pushed to `localhost:5000/my-image:built-with-jib`
-* Runs by calling `java -Xms512m -Xdebug -Xmy:flag=jib-rules -cp app/libs/*:app/resources:app/classes mypackage.MyApp some args`
-* Reproducible
```

Comment:
Why was this removed?

Reply:
Images are reproducible regardless of configuration, so it seemed out of place since this is a configuration-focused example.

Reply:
Okay that makes sense.